app/views/ngos/registrations/edit.html.erb

```diff
@@ -1,34 +1,29 @@
 <h2><%=  t("devise.registrations.edit.title", resource: t("activerecord.models.ngo.one")) %></h2>
 
-<%=  semantic_form_for(resource, as: resource_name,
-                                 url: registration_path(resource_name),
-                                 html: { method: :put }) do |f| %>
-  <%= devise_error_messages! %>
-
-  <%= f.input :email, autofocus: true %>
-  <%= f.input :locale, as: :select, collection: Ngo.locales.keys.to_a %>
-
-  <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
-    <div>Erwarten Bestätigung für: <%= resource.unconfirmed_email %></div>
-  <% end %>
-
-  <%= f.input :password, autocomplete: "off",
-                         hint: "(" + t("devise.registrations.edit.leave_blank_if_you_don_t_want_to_change_it") + ")" %>
-
-  <%= f.input :password_confirmation, autocomplete: "off" %>
-
-  <%= f.input :current_password, autocomplete: "off",
-                                  hint: "(" + t("devise.registrations.edit.we_need_your_current_password_to_confirm_your_changes") + ")" %>
-
-  <%= f.submit t("devise.registrations.edit.update"), class: "btn btn-primary" %>
-<% end %>
-
-<hr/>
-
-<div>
-  <%= button_to t("devise.registrations.edit.cancel_my_account"),
-                 registration_path(resource_name),
-                 data: { confirm: t("devise.registrations.edit.are_you_sure") },
-                 method: :delete,
-                 class: "delete-acount-btn" %>
+<div class="row">
+  <div class="col-sm-12">
+    <%=  semantic_form_for(resource, as: resource_name,
+                                     url: registration_path(resource_name),
+                                     html: { method: :put }) do |f| %>
+      <%= devise_error_messages! %>
+      <div class="col-sm-6">
+        <%= render 'contact_fields', f: f %>
+      </div>
+      <div class="col-sm-6">
+        <%= render 'ngo_fields', f: f %>
+        <%= f.input :locale, as: :select, collection: Ngo.locales.keys.to_a %>
+        <% if devise_mapping.confirmable? && resource.pending_reconfirmation? %>
+          <div>Erwarten Bestätigung für: <%= resource.unconfirmed_email %></div>
+        <% end %>
+        <%= f.input :current_password, autocomplete: "off",
+                                       hint: "(" + t("devise.registrations.edit.we_need_your_current_password_to_confirm_your_changes") + ")" %>
+        <h3><%= t("devise.registrations.edit.cancel_my_account") %> </h3>
```

Comment:
Having this huge headline above the *meant to be* more subtle `cancel account` button is a bit overkill in my opinion. Having the text on the button is enough I think. Or do you see a special benefit in having this? I'd prefer removing the `h3` line.